src/main/java/com/google/api/codegen/config/GapicProductConfig.java

```diff
@@ -475,7 +487,13 @@ private static GapicProductConfig createDummyInstance(
           DiagCollector diagCollector,
           List<InterfaceConfigProto> interfaceConfigProtosList,
           ImmutableMap<String, Interface> protoInterfaces,
-          TargetLanguage language) {
+          TargetLanguage language,
+          ServiceConfig grpcServiceConfig) {
+    if (grpcServiceConfig != null) {
```

Comment:
Indentation seems broken here.

Reply:
If you are referring to the function declaration, gFormat makes it this way. Not a lot I can do about it.

Reply:
Sorry - my bad!